api/src/main/java/org/apache/iceberg/actions/RewriteManifests.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.actions;
+
+import java.util.function.Predicate;
+import org.apache.iceberg.ManifestFile;
+
+/**
+ * An action that rewrites manifests.
+ */
+public interface RewriteManifests extends
+    ConfigurableAction<RewriteManifests, RewriteManifests.Result>,
+    SnapshotUpdate<RewriteManifests, RewriteManifests.Result> {
+
+  /**
+   * Rewrites manifests for a given spec id.
```

Comment:
I think this should mention the default value, which is the table's default spec ID.

Reply:
Fixed.